src/components/crownConfig.vue

```diff
@@ -22,7 +22,7 @@
       :title="$t('Message Flow')"
     />
     <delete-button
```

Comment:
Here's a thought. What if this had a default delete button and action and it could be replaced with a delete-button slot?

Reply:
Ok, the remove-node emit on pool lane could be weird.